.travis.yml

```diff
@@ -5,8 +5,6 @@ python:
   - 3.3
   - 3.4
 env:
-  matrix:
-    - DJANGO=django==1.7.3
```

Comment:
This allows different versions of django to be tested against. It needs to stay.